### PR TITLE
[api] send more info to errbit when IssueTracker#parse_github_issue fails

### PR DESCRIPTION
**Pull request by evanrolfe**
So that we can more easily debug this error based on the response from github:

https://github.com/openSUSE/open-build-service/issues/3349

### CODE REVIEW

File: src/api/app/models/issue_tracker.rb [hunk 1]
```diff
@@ -244,11 +244,15 @@ def parse_github_issues(js)
...
-      return if issue.nil?
+    begin
+      if create
+        issue = Issue.find_or_create_by_name_and_tracker(js["number"].to_s, name)
+      else
+        issue = Issue.find_by_name_and_tracker(js["number"].to_s, name)
+        return if issue.nil?
+      end
+    rescue TypeError => e
+      Airbrake.notify(e, { json_data: js })
```
Comment (mdeniz) [inline src/api/app/models/issue_tracker.rb]: This should still raising the exception? or we are going to fail silently?

Comment (evanrolfe) [inline src/api/app/models/issue_tracker.rb]: good point, I've added `raise` which will re-raise the exception

File: src/api/app/models/issue_tracker.rb [hunk 2]
```diff
@@ -244,13 +244,18 @@ def parse_github_issues(js)
...
+      raise
     end
-
+    binding.pry
```
Comment (DavidKang) [inline src/api/app/models/issue_tracker.rb]: :smile_cat:

Comment (ChrisBr) [inline src/api/app/models/issue_tracker.rb]: Please remove the debug code ;)

File: src/api/app/models/issue_tracker.rb (hunk 1 above)
Comment (ChrisBr) [inline src/api/app/models/issue_tracker.rb]: raise in the next line will do it, right?

File: src/api/app/models/issue_tracker.rb (hunk 2 above)
Comment (evanrolfe) [inline src/api/app/models/issue_tracker.rb]: Ooops! thank god for PR reviews... lol

File: src/api/app/models/issue_tracker.rb (hunk 1 above)
Comment (mdeniz) [inline src/api/app/models/issue_tracker.rb]: I always have the doubt... Is just `raise`? or better
```ruby
 raise e
```

**Pull request merged**